tests/serverspec/chroot.yml

```diff
@@ -1,6 +1,11 @@
 - hosts: localhost
   roles:
-    - ansible-role-unbound
+    - name: ansible-role-unbound
+    - name: reallyenglish.devfsrules
+      when:
+        - ansible_os_family == 'FreeBSD'
+        - unbound_config_chroot is defined
```

Comment:
these two conditionals are always met as `unbound_config_chroot` is defined below.